portfolio/src/main/webapp/script.js

```diff
@@ -41,3 +41,37 @@ async function getLikes(){
   confetti.render();
 }
 
+async function getContacts(){
+  const responseFromServer = await fetch('/hello');
+  const textFromResponse = await responseFromServer.text();
+
+  const commentContainer = document.getElementById('comment');
+  commentContainer.innerText = textFromResponse;
+
+  var confettiSettings = { target: 'my-canvas' };
+  var confetti = new ConfettiGenerator(confettiSettings);
+  confetti.render();
+}
```

Comment:
This function looks like a copy of the one above it (getLikes). Unless I missed it, this function looks unused, so let's remove it.